packages/ai/mcp-stdio/get-environment.ts

```diff
@@ -0,0 +1,37 @@
+export function getEnv(
```

Comment:
rename to `getEnvironment`? (to match file name & prefer full names for the most part)

Reply:
or rename file to `get-env.ts`?